doc/netplan.md

```diff
@@ -190,6 +190,11 @@ Virtual devices
     device's MAC address as a unique identifier rather than a RFC4361-compliant
     Client ID. This has no effect when NetworkManager is used as a renderer.
 
+ ``dhcp-options`` (mapping)
```

Comment:
I really think this needs to be called 'dhcp-overrides', not 'dhcp-options'. 'dhcp-options' is bound to cause confusion with "options" as used in RFC2131.

Reply:
how about dhcp-config

Reply:
I'd be ok with dhcp-config. @cyphermox, thoughts?

Reply:
I just noticed that pull request #43 does something similar, and there it was decided to go with dhcp4-overrides and dhcp6-overrides  (even though networkd doesn't support separate configuration).

Should the decision from that discussion still stand?

Reply:
Indeed it should.

Reply:
I've split dhcp-options into dhcpX-overrides.  I am still working on the docs and tests (probably finish Monday or so), but I figured getting some feedback on this overall approach could be useful.

Also, do you prefer everything to be squashed into a single commit?

Reply:
Marking this resolved since I think I have addressed the concerns here.